Prose/ProseLib/Sources/ConversationFeature/Chat/Chat.swift

```diff
@@ -270,7 +232,17 @@ struct ChatView: NSViewRepresentable {
 
     if !webView.isLoading {
       // TODO: Maybe remove duplicates (see if signpost interval becomes too long)
-      context.coordinator.messagingContext.updateColorScheme(to: self.colorScheme)
+      let styleTheme: StyleTheme = {
+        switch self.colorScheme {
+        case .light:
+          return .light
+        case .dark:
+          return .dark
+        @unknown default:
+          return .light
```

Comment:
I would prefer to leave this as `nil` as I did. If WebKit and macOS introduce a new style, the web view will automatically take the correct theme (assuming we don't use hard-coded colors in `prose-core-views`). Dynamic updates won't work, but at least we won't "override" the new case by `.light`.

Reply:
Done.